templates/repo/graph.tmpl

```diff
@@ -1,7 +1,7 @@
 {{template "base/head" .}}
 <div class="repository commits">
 	{{template "repo/header" .}}
-	<div class="ui container">
+	<div class="ui">
```

Comment:
for something else (don't remember what ) I did a `container.full` which sets the width to "100%"